### DISCUSSION TITLE
Profil MCP i egzamin z domu a polskie ogonki w nazwisku

### DISCUSSION THREAD

**Discussion by zjacek** [AzureCommunityPL/group#158] [Pomoc] [answered]
Hej,

Mam dość stary profil MCP (Microsoft Certification Profile), w którym moje nazwisko jest zapisane bez polskich znaczków.
Do tej pory zdając w różnych ośrodkach nie było z tym problemu, ale teraz będę chciał zapisać się na egzamin z domu.
Powinienem najpierw zaktualizować swoje nazwisko w MCP zanim zapiszę się na egzamin? Czy zapis "e" zamiast "ę" nie powinien stanowić problemu? 
Ktoś się już z takim problemem spotkał?

Prawdę mówiąc nie chce mi się przechodzić tej całej procedury aktualizacji nazwiska w profilu MCP, ale obawiam się, że to może przerwać mój egzamin zanim zacznę ;)

Pozdrawiam
Jacek Zięba

**Comment by eklime** [comment, marked as answer]
Ja bym się nie przejmował. Musisz jedynie wytłumaczyć API, że to polskie litery a w profilu masz zapis "uniwersalny"

**Reply by zjacek** [reply]
To byłoby super. 
Ale co masz na myśli pisząc "wytłumaczyć API"? chodzi o tą osobę, która będzie monitorować egzamin? To ta osoba sprawdza dowód ze zgodnością z MCID ? 

Przy okazji zapytam, to sprawdzanie dzieje się tuż przed egzaminem? Czy dużo wcześniej?

**Reply by smajchrz** [reply]
Sprawdzenie jest chwile przed rozpoczeciem samego egzaminu. Robisz zdjecia dokumentów, siebie i pomieszczenia z ktorego zdajesz.

API = a person in India

**Reply by eklime** [reply]
> API = a person in India

Ooo widzę, że się przyjęło 🙄

**Reply by michal-miziolek-fiedorczuk** [reply]
ja u siebie w profilu MCP mam wpisane bez polskich znaków i powiem szczerze, że ani razu API nie zwrócił na to uwagi ;)
